routemaster/cli.py

```diff
@@ -19,10 +23,38 @@
     type=click.File(encoding='utf-8'),
     required=True,
 )
+@click.option(
+    '-l',
+    '--log-level',
+    help="Logging level.",
+    type=click.Choice((
+        'CRITICAL',
+        'ERROR',
+        'WARNING',
+        'INFO',
+        'DEBUG',
+    )),
+    default='INFO',
+)
 @click.pass_context
-def main(ctx, config_file):
+def main(ctx, config_file, log_level):
     """Shared entrypoint configuration."""
-    config = load_config(yaml.load(config_file))
+    logging.basicConfig(
+        format=(
+            "[%(asctime)s] [%(process)d] [%(levelname)s] "
+            "[%(name)s] %(message)s"
+        ),
+        datefmt="%Y-%m-%d %H:%M:%S %z",
+        level=getattr(logging, log_level),
+    )
+
+    try:
+        config = load_config(yaml.load(config_file))
+    except ConfigError as e:
+        msg = f"Configuration Error: {e}"
```

Comment:
Note that `logging.exception` will already include the details of the exception, so we don't need to include that again in the `msg`.